Library/Homebrew/cmd/audit.rb

```diff
@@ -659,6 +659,25 @@ def audit_line(line, lineno)
     end
   end
 
+  def audit_prefix_has_contents
+    return unless formula.prefix.directory?
+
+    unless Dir.glob("#{formula.prefix}/**/*").any? do |f|
```

Comment:
Wait a minute. @DomT4 you shifted the `end` position when you are adjusting indent.